src/utils/getDatalogQuery.ts

```diff
@@ -120,6 +121,44 @@ const upgradeSelection = (s: Selection, returnNode: string): NewSelection => {
   }
   return selection as NewSelection;
 };
+const upgradeCondition = (c: Condition): NewCondition => {
```

Comment:
We will support the QB version of conditions + selections until it is cutover on that extension.

New version intended to better map to datalog